tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -285,7 +285,32 @@ describe('Boundary Timer Event', () => {
 
   });
 
-  it('it cannot attach an incoming sequence flow on Boundary Timer Events');
+  it('it cannot attach an incoming sequence flow on Boundary Events', function() {
```

Comment:
I don't think we need to double up on the `it` here. We've been following a naming convention that reads as if the text passed to it follows from that word.

```suggestion
  it('cannot attach an incoming sequence flow on Boundary Events', function() {
```